modules/luci-mod-dsl/Makefile

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (C) 2022 Roland Barenbrug <roland@treslong.com>
+#
+# This is free software, licensed under the Apache License, Version 2.0 .
+#
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=LuCI DSL spectrum graph
+LUCI_DEPENDS:=+luci-base
```

Comment:
- We do not need the luci-base dependency here, we only use Javascript?
- We should add `@TARGET_lantiq` or `@TARGET_lantiq_xrx200` dependency, because this only works with this targets?
- We should consider to add dependency to the packages [ltq-vdsl-app](https://github.com/openwrt/openwrt/blob/master/package/network/config/ltq-vdsl-app/Makefile#L79) which is installing `dsl_cpe_pipe.sh`
- I have also seen that [ltq-adsl-app](https://github.com/openwrt/openwrt/blob/master/package/network/config/ltq-adsl-app/Makefile#L88) is installing a similar command. I wonder if this LuCI app does also works with this one.

Reply:
> * We do not need the luci-base dependency here, we only use Javascript?
> * We should add `@TARGET_lantiq` or `@TARGET_lantiq_xrx200` dependency, because this only works with this targets?
> * We should consider to add dependency to the packages [ltq-vdsl-app](https://github.com/openwrt/openwrt/blob/master/package/network/config/ltq-vdsl-app/Makefile#L79) which is installing `dsl_cpe_pipe.sh`
> * I have also seen that [ltq-adsl-app](https://github.com/openwrt/openwrt/blob/master/package/network/config/ltq-adsl-app/Makefile#L88) is installing a similar command. I wonder if this LuCI app does also works with this one.

Per item:

- not sure. the DSL charts do depend on a luci foundation being available. Without Luci being build it is not going the work
- on Lantiq, this make sense. It is all related to xDSL and that's only supported on Lantiq based platforms.
- correct
- should work as well. Personally not having access to ADSL hence  bit difficult to test.

On dependencies, just need to figure out how to put this in but should be able to manage this.

Reply:
Makefile comment has been soved with https://github.com/openwrt/luci/pull/5895/commits/def03e6216f7e9727eacfdb7c5c64aa94a686f74